cabal.project

```diff
@@ -11,3 +11,8 @@ packages:
   proto-lens-setup
   proto-lens-tests
   proto-lens-tests-dep
+
+source-repository-package
```

Comment:
I understand why you want to make this change, but I'd prefer to have the new version of ghc-source-gen pushed to hackage instead. 

The rest of the changes are good to go, if you want them merged first.

Reply:
I have removed these lines as requested.